Add mynth to companies

graxiemoxie opened the pull request.

graxiemoxie committed:
Add Mynth Logo

graxiemoxie committed:
Add Mynth company logo data to JSON

katomm reviewed (changes requested):
Hello, please note that the field knownFor must be set to at least one product/service or tool that is listed on either the Cardano Showcase https://developers.cardano.org/showcase or Builder Tools https://developers.cardano.org/tools.

Details on all the requirements: https://cardano.org/docs/tutorial/add-company

graxiemoxie committed:
Update knownFor field for Mynth company

Updated the knownFor field for Mynth to match the product listed on the [Cardano Showcase](https://github.com/cardano-foundation/developer-portal/blob/fa96fbd9af1143a4bec75f881f4fa98c8bce5412/src/data/showcases.js#L1368).

katomm reviewed (changes requested):
Sorry for my late follow-up: You cannot add a product, service or tool. 

--> Mynth is the product, this list is for companies. (I assume "Progressive IT s.r.o.") and it must not go to the same URL like the showcase entry.

graxiemoxie committed:
Update company name and details for Mynth

graxiemoxie commented:
Hi @katomm, thanks for your feedback! I’ve updated the company name and details as requested in the latest commit. Please let me know if any further changes are needed. Thanks!

katomm reviewed (changes requested):
It still links to the same website.  Please see https://cardano.org/docs/get-involved/add-company/#website-requirement for details.
